src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -131,68 +132,74 @@ private void WriteRuntimeConfig(ProjectContext projectContext)
 
         private void AddFrameworks(RuntimeOptions runtimeOptions, ProjectContext projectContext)
         {
+            runtimeOptions.Tfm = TargetFramework;
+
+            List<RuntimeConfigFramework> frameworks = new List<RuntimeConfigFramework>();
```

Comment:
nit: var since typoe is obvious.
```suggestion
            var frameworks = new List<RuntimeConfigFramework>();
```